src/applications/disability-benefits/all-claims/tests/config/781.unit.spec.js

```diff
@@ -0,0 +1,130 @@
+import { expect } from 'chai';
+
+import { formConfig781, formConfig781a } from '../../config/781';
+
+describe('781/781a incident form config iterators', () => {
+  describe('781 incident form config', () => {
+    it('should return a config object', () => {
+      expect(formConfig781(1)).to.be.an('object');
+    });
+
+    it('should return three incident date page config objects', () => {
+      const config = formConfig781(3);
+
+      expect(config).to.haveOwnProperty('incidentDate0');
```

Comment:
This is all pretty repetitive, you might want to consider creating a function that can run these expect statements on a specified iteration, instead of copying and paste all of them for each of the three.